tools-reference/head-and-tail/text.xml

```diff
@@ -32,8 +32,8 @@ file. The <c>-n</c> argument specifies how many lines to display.
 </p>
 
 <p>
-To specify "the last five lines", use <c>tail -n5</c>. To specify "all
-but the first five lines", use <c>tail -n+5</c>.
+To specify "the last five lines", use <c>tail -n 5</c>. To specify "all
```

Comment:
I personally prefer `-n5` over `-n 5` but I don't care either way. However, we should maintain some consistency if possible. The `cut` page also omits the whitespace. If we're gonna go with `-n 5`, we should fix `cut` to insert a whitespace as well.

Reply:
POSIX Utility Syntax Guidelines (Guideline 6) say that the whitespace should be there:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html#tag_12_02

I have fixed `cut` as well.